docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,62 @@ discretize
+mesh = discretize(poly, FIST())
+plot(plot(poly), plot(mesh))
+savefig("plot2.svg")
+Markdown.parse("![Plot](plot2.svg)")
+```
+```@eval
+using Plots
+using Meshes
+using Markdown
+gr(size = (500,300))
+poly = PolyArea(Point{2,Float32}[Point(0.18142937f0, 0.54681134f0), Point(0.38282228f0, 0.107781954f0), Point(0.43220532f0, 0.013640274f0), Point(0.48068276f0, 0.019459315f0), Point(0.48322055f0, 0.11583236f0), Point(0.46696007f0, 0.2230227f0), Point(0.48184678f0, 0.2656454f0), Point(0.45998818f0, 0.2784367f0), Point(0.4168235f0, 0.2190962f0), Point(0.4124987f0, 0.21208182f0), Point(0.39593673f0, 0.2520411f0), Point(0.44333926f0, 0.28375763f0), Point(0.4978224f0, 0.3981428f0), Point(0.7703431f0, 0.20181546f0), Point(0.7612364f0, 0.33008572f0), Point(0.9856581f0, 0.2215304f0), Point(0.99374324f0, 0.3353423f0), Point(0.9688778f0, 0.38663587f0), Point(0.59554976f0, 0.655444f0), Point(0.59496254f0, 0.58492756f0), Point(0.27641845f0, 0.656314f0), Point(0.3242084f0, 0.6072907f0), Point(0.42408508f0, 0.49353212f0), Point(0.20984341f0, 0.59003067f0), Point(0.18142937f0, 0.54681134f0)], [])
+mesh = discretize(poly, FIST())
+plot(plot(poly), plot(mesh))
+savefig("plot3.svg")
+Markdown.parse("![Plot](plot3.svg)")
+```
+```@eval
+using Plots
+using Meshes
+using Markdown
+gr(size = (500,300))
+poly = PolyArea(Point{2,Float32}[Point(0.20974362f0, 0.11143557f0), Point(0.24619843f0, 0.019633872f0), Point(0.2525517f0, 0.10491638f0), Point(0.2605699f0, 0.03590731f0), Point(0.2794662f0, 0.009585844f0), Point(0.54061747f0, 0.2523168f0), Point(0.5210726f0, 0.27176696f0), Point(0.55140936f0, 0.29682124f0), Point(0.5437957f0, 0.24153952f0), Point(0.6285009f0, 0.21651964f0), Point(0.6449456f0, 0.2147006f0), Point(0.59213245f0, 0.26681182f0), Point(0.8229673f0, 0.74253976f0), Point(0.65908766f0, 0.56879747f0), Point(0.5620604f0, 0.53223926f0), Point(0.52954125f0, 0.5394655f0), Point(0.23878936f0, 0.1011955f0), Point(0.22548631f0, 0.09457563f0), Point(0.2759589f0, 0.71809477f0), Point(0.5383939f0, 0.85454345f0), Point(0.6496192f0, 0.9248424f0), Point(0.6248543f0, 0.8470848f0), Point(0.749698f0, 0.8314047f0), Point(0.75198966f0, 0.9489501f0), Point(0.7027816f0, 0.92791766f0), Point(0.7210894f0, 0.8686051f0), Point(0.65642047f0, 0.9434703f0), Point(0.6423689f0, 0.9363239f0), Point(0.549272f0, 0.9676014f0), Point(0.23183312f0, 0.71278405f0), Point(0.20974362f0, 0.11143557f0)], [])
+mesh = discretize(poly, FIST())
+plot(plot(poly), plot(mesh))
+savefig("plot4.svg")
+Markdown.parse("![Plot](plot4.svg)")
```

Comment:
Why do you do this only here? If it's not required, you should not save the result to a file.

Reply:
@serenity4  I have made the changes. Please review.